--- guile-sdl-0.4.3/src/sdlgfx.c.orig	2013-03-18 18:24:55.000000000 +0100
+++ guile-sdl-0.4.3/src/sdlgfx.c	2013-04-24 16:57:08.435201056 +0200
@@ -62,7 +62,7 @@
  * gfxPrimitives
  */
 
-#include "SDL_gfx/SDL_gfxPrimitives.h"
+#include "SDL/SDL_gfxPrimitives.h"
 
 
 PRIMPROC
@@ -688,7 +688,7 @@
  * rotozoom
  */
 
-#include "SDL_gfx/SDL_rotozoom.h"
+#include "SDL/SDL_rotozoom.h"
 
 
 PRIMPROC
@@ -824,7 +824,7 @@
  * framerate
  */
 
-#include "SDL_gfx/SDL_framerate.h"
+#include "SDL/SDL_framerate.h"
 
 static long fpsmgr_tag;
 
@@ -943,7 +943,7 @@
  * RGBA extras
  */
 
-#include "SDL_gfx/SDL_gfxBlitFunc.h"
+#include "SDL/SDL_gfxBlitFunc.h"
 
 
 PRIMPROC
@@ -997,7 +997,7 @@
  * image filter
  */
 
-#include "SDL_gfx/SDL_imageFilter.h"
+#include "SDL/SDL_imageFilter.h"
 
 PRIMPROC
 (imfi_mmx_p, "imfi-mmx?", 0, 1, 0,
--- guile-sdl-0.5.0/src/Makefile.am.orig	2013-07-10 12:17:16.000000000 +0200
+++ guile-sdl-0.5.0/src/Makefile.am	2013-07-24 17:26:51.596567438 +0200
@@ -18,7 +18,7 @@
 ## Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
 ## Boston, MA  02110-1301  USA
 
-SUBDIRS = SDL_gfx k
+SUBDIRS = k
 
 AM_CPPFLAGS = $(SDL_CFLAGS)
 AM_CFLAGS = $(NORMAL_CFLAGS)
@@ -85,13 +85,13 @@
 
 gfx_la_SOURCES  = sdlgfx.c $(headers)
 gfx_la_LDFLAGS  = $(modflags)
-gfx_la_LIBADD   = SDL_gfx/libSDL_gfx.la \
-                  SDL_gfx/libimfi.la
+gfx_la_LIBADD   = -lSDL_gfx
+gfx_la_CPPFLAGS = -I/usr/include/SDL
 
 SUFFIXES = .c
 
 gx		= $(top_srcdir)/build-aux/guile-baux/gbaux-do
-snarfcppopts	= $(DEFS) $(DEFAULT_INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS)
+snarfcppopts	= $(DEFS) $(DEFAULT_INCLUDES) $(AM_CPPFLAGS) $(CPPFLAGS) -I/usr/include/SDL
 
 .c.x:
 	$(gx) c2x -o $@ $< -- $(snarfcppopts)
--- guile-sdl-0.5.0/configure.ac.orig	2013-07-24 17:25:26.483237676 +0200
+++ guile-sdl-0.5.0/configure.ac	2013-07-24 17:27:34.633232297 +0200
@@ -128,7 +128,6 @@
 AC_CONFIG_FILES([
   Makefile
   src/Makefile
-  src/SDL_gfx/Makefile
   src/k/Makefile
   doc/Makefile
   test/Makefile
